src/ft2font_wrapper.cpp

```diff
@@ -507,7 +511,7 @@ static int PyFT2Font_init(PyFT2Font *self, PyObject *args, PyObject *kwds)
 exit:
     Py_XDECREF(data);
 
-    return 0;
+    return ret;
```

Comment:
I usually prefer `return PyErr_Occurred() ? -1 : 0;` rather than tracking the return value separately (the cost should be negligible compared to e.g. the filesystem access needed for loading the font), but won't absolutely insist on it.

Reply:
I just forgot that this existed.